src/components/Main.jsx

```diff
@@ -39,27 +41,53 @@ class MainComponent extends Component {
     }
   }
 
+  printPDF = () => {
+    //for temp loading
+    document.body.classList.add("cursor-wait")
```

Comment:
This is not necessary. Use tailwind spin class `https://tailwindcss.com/docs/animation#spin`